### DISCUSSION TITLE
The project is working locally, but classes are missing once I run it on github pages

### DISCUSSION THREAD

**Discussion by Amit-Sucher** [community/community#124678] [Programming Help]
### Select Topic Area

Question

### Body

https://github.com/ExcaliburFRC/ExcaliScout
I have this project, when I run this locally it works, but when I run it on github pages it does not work, any possible reason for that?

I also see on the site that it is missing a lot of classes
![image](https://github.com/community/community/assets/108475013/d5e22153-e05d-412e-8e63-06916a108571)

**Comment by LiteBrite82** [comment]
Your file structure is a little off. 
- public/index.html has been altered from its original state. it needs to be left alone. 
- its best practice to use src/App.js just for routing; all that code for the "scouting" path should be in its own component. 

The code is hard to follow because of these issues; makes it hard to troubleshoot. 
I get the following errors when I deploy your built code: 
![Screenshot 2024-05-30 at 4 19 16 PM](https://github.com/community/community/assets/163985914/76e43329-b04e-48f3-a2ea-6575271053ab)

The main problem: File references - somewhere in the build, an href is pointing to an incorrect directory. Or it could be bable messing it up, or an incorrect react import; its hard to tell. 

- Create-React-App is no longer being maintained; its much better to use a framework like Vite instead.
